apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/WalletDetailsScreen.tsx

```diff
@@ -233,7 +233,7 @@ export const WalletDetailsScreen = () => {
   ])
 
   React.useEffect(() => {
-    if (showRestoreWalletInfoModal) showModalTipsPassword()
+    if (showRestoreWalletInfoModal === true) showModalTipsPassword()
```

Comment:
```suggestion
    if (showRestoreWalletInfoModal) showModalTipsPassword()
```

Reply:
Not sure why this was complaining for me, saying `showRestoreWalletInfoModal` was `any` and required a specific check, despite it being a boolean.

Reply:
I have weird messages on it too sometimes with VSCode + eslint, then I close (like windows) I close VSCode and open it again, it stops complaining, it looks this rule for booleans have has come conflict with VSC or something is not working between them.